CMakeLists.txt

```diff
@@ -1073,6 +1073,27 @@ set(LOVE_SRC_MODULE_WINDOW
 source_group("modules\\window" FILES ${LOVE_SRC_MODULE_WINDOW_ROOT})
 source_group("modules\\window\\sdl" FILES ${LOVE_SRC_MODULE_WINDOW_SDL})
 
+#
+# Love scripts
+# Useful while developing
+#
+
+set(LOVE_SRC_SCRIPTS
+    src/scripts/nogame.lua
+)
+
+find_program(LOVE_LUA_PROGRAM NAMES lua luajit lua54 lua53 lua52 lua51)
+if(LOVE_LUA_PROGRAM)

```

Comment:
I plan to eventually delete auto.lua if we can find a way to stop the no-game code from needing it (similar to all the other in-tree lua files in love's code, which no longer need auto.lua). So I'm not sure about adding stuff like this and the auto.lua changes